README.md

```diff
@@ -41,6 +41,7 @@ More examples in testcases/ directory.
 
 To see all the tests passing execute below commands.
 
+### Cluster Tests
 ```
 git clone https://github.com/kyma-incubator/octopus
```

Comment:
octopus and grafana should be moved to respective sections. Cluster tests needs only bare k3s cluster.